source/NVDAObjects/UIA/__init__.py

```diff
@@ -233,7 +236,9 @@ def _getControlFieldForObject(self, obj,isEmbedded=False,startOfNode=False,endOf
 		field["description"] = obj.description
 		field["level"] = obj.positionInfo.get("level")
 		if role == controlTypes.ROLE_TABLE:
-			field["table-id"] = 1 # FIXME
+			tableID=VARIANT()
+			obj.UIAElement._IUIAutomationElement__com_GetCurrentPropertyValue(UIAHandler.UIA_RuntimeIdPropertyId,byref(tableID))

```

Comment:
I don't think this is being used? And if it is, why do we need to get it this way?